client/app/components/dynamic-table/image-cell/index.js

```diff
@@ -0,0 +1,66 @@
+import { isUndefined } from 'underscore';
+
+const hasOwnProperty = Object.prototype.hasOwnProperty;
+
+function trim(str) {
+  return str.replace(/^\s+|\s+$/g, '');
+}
+
+function processTags(template, data, defaultColumn) {
+  return template.replace(/{{\s*([^\s]+)\s*}}/g, (match, column) => {
+    if (column === '@') {
+      column = defaultColumn;
+    }
+    if (hasOwnProperty.call(data, column) && !isUndefined(data[column])) {
+      return data[column];
+    }
+    return match;
+  });
+}
```

Comment:
I'm not sure if we gain anything by it, but maybe use [`_.template`](http://underscorejs.org/#template)?

Reply:
I think `_.template` is too much for such a simple thing.